src/ProductFeedStatus.php

```diff
@@ -90,9 +99,61 @@ public static function set( $state ) {
 	 * @return void
 	 */
 	public static function deregister() {
-
+		$properties_to_persist = array(
```

Comment:
In case the feed is deregistered we should not send the count and time info - we should use the default values. This means that this logic can be eliminated.